common/src/boot_status.rs

```diff
@@ -64,7 +64,7 @@ pub enum RomBootStatus {
 
     // Cold Reset Statuses
     ColdResetStarted = COLD_RESET_BOOT_STATUS_BASE,
-    ColdResetComplete = COLD_RESET_BOOT_STATUS_BASE + 1,
+    ColdResetComplete = UPDATE_RESET_BOOT_STATUS_BASE - 1,
```

Comment:
What is the purpose of this change? Does this collide with UpdateResetLoadManifestComplete ?

Reply:
This is being done to not have ColdResetStarted and ColdResetComplete differ by a single bit. With this change, ColdResetComplete will be 320 (UPDATE_RESET_BOOT_STATUS_BASE - 1) and UpdateResetStarted will be 321 (UPDATE_RESET_BOOT_STATUS_BASE). UpdateResetLoadManifestComplete is 322.